lenses/nginx.aug

```diff
@@ -42,11 +45,20 @@ let block_re_all = block_re | "if" | "location" | "geo" | "map"
 (* View: simple
      A simple entry *)
 let simple =
-     let kw = /[A-Za-z0-9_.:-]+/ - block_re_all
+     let kw = word - block_re_all
   in let mask = [ label "mask" . Util.del_str "/" . store Rx.integer ]
   in let sto = store /[^ \t\n;][^;]*/ . Sep.semicolon
   in [ Util.indent . key kw . mask? . Sep.space . sto . (Util.eol|Util.comment_eol) ]
 
+(* View: server
+     A simple server entry *)
+let server =
+  [ Util.indent . label "@server" . Util.del_str "server"
+  . [ Sep.space . label "@address" . store word ]
+  . [ Sep.space . key word . Sep.equal . store word ]*
```

Comment:
The equal sign and word are optional, see http://nginx.org/en/docs/http/ngx_http_upstream_module.html for lots of examples.